packages/rich-text/src/useOnValueChanged.ts

```diff
@@ -31,8 +31,13 @@ export const useOnValueChanged = ({ editorId, handler, skip, onSkip }: OnValueCh
   const onChange = useMemo(
     () =>
       debounce((document: unknown) => {
-        const contentfulDocument = toContentfulDocument({ document, schema });
-        const cleanedDocument = removeInternalMarks(contentfulDocument);
+        const contentfulDocument = toContentfulDocument({
+          // eslint-disable-next-line -- parameter type is not exported @typescript-eslint/no-explicit-any
+          document: document as any,
```

Comment:
why does this need to be any now?

Reply:
I wrote about this in the description. The old build of `contentful-slatejs-adapter` didn’t properly expose its types so nobody ever checked that the types make sense. Updating to the new version of `contentful-slatejs-adapter` fixed the types but now all the errors pop up here. Fixing the types is out of scope of this change.
E.g. this code calls two functions from `contentful-slatejs-adapter` and passes the return value of the first to the second and somehow the types are incompatible.